minio/minio
# Save bootstrap trace events in a circular buffer

## Description
These saved bootstrap events will be published to new trace event subscribers if they subscribe to bootstrap type events

## Motivation and Context
Tracing bootstrap events requires that we coordinate invocation of `mc admin trace --call bootstrap ...` with the MinIO cluster startup. This is inconvenient and doesn't guarantee that `mc admin trace` connects to the MinIO server soon enough to receive all the bootstrap events.

## How to test this PR?
1. Start a MinIO cluster
2. After it has started up, run `mc admin trace --call bootstrap ALIAS`
You should see bootstrap events that occurred at the time of startup.
3. You can run this command multiple times and continue to receive these events for a period of 24 hrs since the last bootstrap event.

## Types of changes

## Checklist:
